mail/views_test.py

```diff
@@ -229,13 +231,62 @@ def test_automatic_email_fail(self):
             "email_subject": self.request_data['email_subject'],
             "email_body": self.request_data['email_body'],
             "sender_name": full_name(self.staff),
+            "staff_user": self.staff,
         }
 
         assert sorted(SentAutomaticEmail.objects.filter(
             user__email__in=self.email_results, automatic_email=self.automatic_email,
         ).values_list('user__email', flat=True)) == sorted(success_emails)
 
 
+class AutomaticEmailViewTests(APITestCase, MockedESTestCase):
+    """
+    AutomaticEmailViewTests
+    """
+
+    def setUp(self):
+        self.staff_user = UserFactory.create()
+        self.program = ProgramFactory.create()
+        Role.objects.create(
+            user=self.staff_user,
+            program=self.program,
+            role=Staff.ROLE_ID,
+        )
+        self.url = reverse('automatic_email_api')
+        self.client.force_login(self.staff_user)
+
+    def test_normal_users_cant_get(self):
+        """
+        normal users shouldnt be able to get any data
+        """
+        normal_user = UserFactory.create()
+        self.client.force_login(normal_user)
+        response = self.client.get(self.url)
+        assert response.status_code == 403
```

Comment:
please use the variable in rest_framework.status and not the number directly (and the same for all the other tests)